script/__snapshots__/verify-changes.test.ts.snap

```diff
@@ -0,0 +1,849 @@
+--- a/README.md
++++ b/README.md
+@@ -5,57 +5,20 @@
+ <p align=\\"center\\">
+ 	<!-- prettier-ignore-start -->
+ 	<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+-<a href=\\"#contributors\\" target=\\"_blank\\"><img alt=\\"All Contributors: 36 🤝\\" src=\\"https://img.shields.io/badge/all_contributors-36_🤝-21bb42.svg\\" /></a>
++<a href=\\"#contributors\\" target=\\"_blank\\"><img alt=\\"All Contributors: 36 👪\\" src=\\"https://img.shields.io/badge/all_contributors-36_👪-21bb42.svg\\" /></a>
+ <!-- ALL-CONTRIBUTORS-BADGE:END -->
+ 	<!-- prettier-ignore-end -->
+ 	<a href=\\"https://codecov.io/gh/JoshuaKGoldberg/create-typescript-app\\" target=\\"_blank\\"><img alt=\\"Codecov Test Coverage\\" src=\\"https://codecov.io/gh/JoshuaKGoldberg/create-typescript-app/branch/main/graph/badge.svg\\"/></a>
+-	<a href=\\"https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/.github/CODE_OF_CONDUCT.md\\" target=\\"_blank\\"><img alt=\\"Code of Conduct: Enforced 🤝\\" src=\\"https://img.shields.io/badge/code_of_conduct-enforced_🤝-21bb42\\" /></a>
+-	<a href=\\"https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/LICENSE.md\\" target=\\"_blank\\"><img alt=\\"License: MIT 📝\\" src=\\"https://img.shields.io/badge/license-MIT_📝-21bb42.svg\\"></a>
+-	<a href=\\"https://github.com/sponsors/JoshuaKGoldberg\\" target=\\"_blank\\"><img alt=\\"Sponsor: On GitHub 💸\\" src=\\"https://img.shields.io/badge/sponsor-on_github_💸-21bb42.svg\\" /></a>
+-	<img alt=\\"Style: Prettier 🧹\\" src=\\"https://img.shields.io/badge/style-prettier_🧹-21bb42.svg\\" />
+-	<img alt=\\"TypeScript: Strict 💪\\" src=\\"https://img.shields.io/badge/typescript-strict_💪-21bb42.svg\\" />
++	<a href=\\"https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/.github/CODE_OF_CONDUCT.md\\" target=\\"_blank\\"><img alt=\\"Contributor Covenant\\" src=\\"https://img.shields.io/badge/code_of_conduct-enforced-21bb42\\" /></a>
++	<a href=\\"https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/LICENSE.md\\" target=\\"_blank\\"><img alt=\\"License: MIT\\" src=\\"https://img.shields.io/github/license/JoshuaKGoldberg/create-typescript-app?color=21bb42\\"></a>
++	<a href=\\"https://github.com/sponsors/JoshuaKGoldberg\\" target=\\"_blank\\"><img alt=\\"Sponsor: On GitHub\\" src=\\"https://img.shields.io/badge/sponsor-on_github-21bb42.svg\\" /></a>
++	<img alt=\\"Style: Prettier\\" src=\\"https://img.shields.io/badge/style-prettier-21bb42.svg\\" />
++	<img alt=\\"TypeScript: Strict\\" src=\\"https://img.shields.io/badge/typescript-strict-21bb42.svg\\" />
+ 	<img alt=\\"npm package version\\" src=\\"https://img.shields.io/npm/v/create-typescript-app?color=21bb42\\" />
++	<a href=\\"#contributors\\" target=\\"_blank\\"><img alt=\\"All Contributors: 36 🤝\\" src=\\"https://img.shields.io/badge/all_contributors-36_🤝-21bb42.svg\\" /></a>
++	<img alt=\\"Sponsor: On GitHub 💸\\" src=\\"https://img.shields.io/badge/sponsor-on_github_💸-21bb42.svg\\" />
```

Comment:
[Non-actionable] Aha, a diff I wasn't expecting!

Filed #1088 to track fixing this unexpected difference. No action needed as part of this PR.